app/views/questions/_form.html.erb

```diff
@@ -11,14 +11,17 @@
     </div>
   <% end %>
 
-  <div class="field">
+  <div class="field hidden">
     <%= form.label :body %>
     <%= form.text_field :body, id: :question_body %>
   </div>
 
+
+  <div id="editor-container"></div>
+
   <div class="field">
     <%= form.label :for_course %>
-    <%= form.select(:course_name, Course.all.map { |c| [c.name, c.id] } ) %>
+    <%= form.select(:course_name, current_user.courses_as_instructor.map { |c| [c.name, c.name] } ) %>
```

Comment:
Fix here. A user should only be prompted for course names for which they have access to